tango.pc.cmake

```diff
@@ -2,10 +2,11 @@ prefix = @CMAKE_INSTALL_PREFIX@
 exec_prefix = ${prefix}
 includedir = ${prefix}/include
 libdir = ${prefix}/@CMAKE_INSTALL_LIBDIR@
+devices_dir = ${prefix}/@CMAKE_INSTALL_LIBDIR@/tango
```

Comment:
We need more info from Frederic. I believe `${prefix}/@CMAKE_INSTALL_LIBDIR@/tango` is what is used by default. `dserver` sounds for me as a admin device... anyway lets wait for Frederic's comment